src/components/FeaturedEventDialog.tsx

```diff
@@ -23,13 +23,9 @@ interface FormDataType {
 	email: string;
 	phone: string;
 }
-const FeaturedEventDialog: React.FC<Props> = ({
-	isOpen,
-	onClose,
-	onOpen,
-	training,
-}) => {
+const FeaturedEventDialog: React.FC<Props> = ({ training }) => {
 	const [numPeople, setNumPeople] = useState(1);
```

Comment:
Coming back to this number of people, use [derived state](https://www.linkedin.com/pulse/why-you-should-use-derive-state-react-better-rohan-vania/)

Reply:
I still don't understand how you mean here. 
At the start of the opening of the booking dialog, the number of people is one by default. The numOfPeople changes based on how many people/ participants are to book for a training.

Could you emphasize how the derive state can be added here. I use this is in about 10 places to decide the new price of the training, people attending and so on.

Reply:
I'll refactor it later after the branch is merged

Reply:
Ok. No problem. I'll keep an eye on it to see the refactored version.